types/stripe-node/index.d.ts

```diff
@@ -4238,14 +4241,29 @@ declare namespace StripeNode {
 
         interface ISubscriptionListOptions extends IListOptionsCreated {
             /**
+             * The billing mode of the subscriptions to retrieve.
+             */
+            billing?: "charge_automatically" | "send_invoice";
+
+            /**
              * The ID of the customer whose subscriptions will be retrieved
              */
             customer?: string;
 
             /**
+             * A limit on the number of objects to be returned. Limit can range between 1 and 100 items, and the default is 10 items.
+             */
+            limit?: number;
```

Comment:
This is already inherited from `IListOptions`

Reply:
Remove that commit 👌